Techion.rte/Devices/Shared/Scripts/PulseShot.lua

```diff
@@ -59,6 +64,7 @@ function PulsarDissipate(self, inverted)
 		self.explosion.Vel = offset;
 		MovableMan:AddParticle(self.explosion);
 	end
+	self.lastVel = Vector(self.Vel.X, self.Vel.Y);
```

Comment:
Did you ever try using OnCollideWithMO for this? Or didn't wanna deal with it yet?

Reply:
I don't think it ever worked.

Reply:
Odd, I'm sure other people have used it successfully. If you do feel like trying it out, please lemme know if it doesn't work.